src/main/java/wolox/training/exceptions/responses/BookNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.exceptions.responses;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
I think you can remove this line because you're handling it with the ExceptionHandler 😄

Reply:
Okidoki, thank you so much :smile: